README.md

```diff
@@ -56,6 +56,40 @@ to your app branch.
 
 See [project boilerplate!](https://github.com/plotly/dash-sample-apps#project-boilerplate)
 
+### Notes on adding a new Dash for R app
+
+Contributing an app written with Dash for R is very similar to the steps outlined above. 
+
+1. Make the folder have the _exact same_ name as the Dash app name.
+
+2. Ensure that the file containing your app code is named `app.R`.
+
+3. The `Procfile` should contain `web: R -f /app/apps/"$DASH_APP_NAME"/app.R`.
```

Comment:
3. Contents of Procfile should be:
```
web: R -f /app/apps/"$DASH_APP_NAME"/app.R
```

So that it is easier to copy and paste..!!

Reply:
💡